src/app/shared/helpers/test-start-time.ts

```diff
@@ -0,0 +1,46 @@
+import * as moment from 'moment';
+
+export const PRESS_TIME_TO_ENABLE_EDIT = 10000;
+
+// This function is required for updating incorrect dates of test
+// Is mainly used in the office page of Delegated test journies
+
+export function getNewTestStartTime(inputDate: string, startDateTime: string): string {
```

Comment:
Assuming this lives in DES3, although think a TODO should be made to refactor this method to use the DateTime service

Reply:
story created (MES-7239)